setup.py

```diff
@@ -1,87 +1,41 @@
 import os
-import sys
+from codecs import open
 
 from setuptools import find_packages, setup
 
 
-version = '0.3.1'
+BASE_DIR = os.path.abspath(os.path.dirname(__file__))
+VERSION = __import__('didadata').__version__
 
 
-if sys.argv[-1] == 'publish':
-    os.system('python setup.py sdist upload')
-    os.system('python setup.py bdist_wheel upload')
-    print('You probably want to also tag the version now:')
-    print('  git tag -a %s -m "version %s"' % (version, version))
-    print('  git push --tags')
-    sys.exit()
-
-
-def read(*parts):
-    filename = os.path.join(os.path.dirname(__file__), *parts)
-    with open(filename) as fp:
-        return fp.read()
-
-
-tests_require = [
-    'coverage',
-    'pytest',
-    'pytest-cov',
-    'pytest-pep8',
-    'pytest-flakes',
-    'pytest-isort',
-    'pytest-django',
-    'factory-boy',
-]
-
-
-docs_requires = [
-    'sphinx',
-    'sphinx_rtd_theme'
-]
+with open(os.path.join(BASE_DIR, 'README.rst'), encoding='utf-8') as f:
+    long_description = f.read()
 
 
 setup(
     name='django-didadata',
+    version=VERSION,
     description='A Django app to collecto numeric data.',
-    long_description=read('README.rst'),
-    version=version,
-    license='BSD',
```

Comment:
Why was the line removed?

Reply:
I have put long_description and version on the top and forgot the license... will be added.